pkg/utils/util.go

```diff
@@ -91,3 +97,18 @@ func CmpFloat(p1, p2 float64) int32 {
 	}
 	return 1
 }
+
+func GetFileMd5(file string) (string, error) {
```

Comment:
use fsnotify to implement it.

please refer to:
https://github.com/gocrane/crane/blob/main/pkg/recommendation/manager.go#L78